File: SHELBY_COMPANY_LIMITED/Web/python/conexao.py
# Instalar o Python 3
# pip install pymysql

import pymysql

def conectar():
    hostname = "localhost"
    bancodedados = "agendamento"
    usuario = "root"
    senha = ""

    try:
        conexao = pymysql.connect(
            host=hostname,
            user=usuario,
            password=senha,
            database=bancodedados
        )

        return True, conexao
    
    except pymysql.Error as erro:
        print("Falha ao conectar ao banco de dados:", erro)
        return False, None

File: SHELBY_COMPANY_LIMITED/Web/conexao.php
<?php

$host = "localhost";
$user = "root";
$db_password = "";
$db_name = "agendamento";

try {
    $conn = new PDO("mysql:host=$host; dbname=" . $db_name, $user, $db_password);
    //echo "conectou";
} catch (PDOException $err) {
    echo "Erro conexão" . $err->getMessage();
}
